package.json

```diff
@@ -213,11 +213,13 @@
     "release": "semantic-release",
     "start:testing": "NODE_ENV=testing webpack-dev-server --config ./webpack/webpack.dev.js --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
     "start": "lingui compile && NODE_ENV=development webpack-dev-server --config ./webpack/webpack.dev.js --hot --progress --colors --content-base ./dist --host 0.0.0.0 --port $npm_package_config_port",
-    "test:integration:local": "npx cypress open",
     "test:integration:plugins:setup": "[ -z \"${npm_config_externalplugins}\" ] || rsync -aH ${npm_config_externalplugins}/tests/ ./tests/",
+    "test:integration:local": "PLUGINS_PATH=\"${npm_config_externalplugins}\" ./scripts/run-integration-tests -hnk",
     "test:integration:rerun": "PLUGINS_PATH=\"${npm_config_externalplugins}\" ./scripts/ci/rerun-integration-tests.sh",
     "test:integration": "PLUGINS_PATH=\"${npm_config_externalplugins}\" ./scripts/run-integration-tests",
+    "test:system:plugins:setup": "[ -z \"${npm_config_externalplugins}\" ] || rsync -aH ${npm_config_externalplugins}/system-tests/ ./system-tests/",
```

Comment:
I dont think we need this, `test:integration` will do it.

Reply:
Not if you run it with the no build option